src/np_client/date_format.rs

```diff
@@ -35,15 +35,16 @@ pub mod common_date_format {
             .or(NaiveDateTime::parse_from_str(&s, FORMAT2))
             .or(NaiveDateTime::parse_from_str(&s, FORMAT3))
             .or(NaiveDateTime::parse_from_str(&s, FORMAT4))
-            .or(NaiveDate::parse_from_str(&s, FORMAT5).map(|date| date.and_time(NaiveTime::default())))
+            .or(NaiveDate::parse_from_str(&s, FORMAT5)
+                .map(|date| date.and_time(NaiveTime::default())))
             .map(|date| Some(date))
             .map_err(serde::de::Error::custom);
         res
     }
 }
 
 pub mod np_date_format {
-    use chrono::{DateTime, FixedOffset, NaiveDateTime, TimeZone, Utc};
+    use chrono::NaiveDateTime;
     use serde::{self, Deserialize, Deserializer, Serializer};
 
     const FORMAT: &'static str = "%d-%m-%Y %H:%M:%S";
```

Comment:
Constants have static lifetime by default and it in unnecessary to declare it.

```suggestion
    const FORMAT: &str = "%d-%m-%Y %H:%M:%S";
```

This suggestion was shown to me in my VSCode because I have enabled `clippy` integration with `rust-analyzer`.
If you are using `rust-analyzer` together with `VSCode`, I suggest to add this to your `settings.json` config.
```
"rust-analyzer.checkOnSave.command": "clippy",
"rust-analyzer.rustfmt.enableRangeFormatting": true
```

Reply:
Thank you! I should have done it long time ago 😅